content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,431 @@
+posts should be sorted, by modifying the :guilabel:`Default Sort` field. Change the
+:guilabel:`Privacy` setting to change which users will have access to the forum.
+
+Below these settings, there is space to add a description for the forum. This description will be
+visible on the website where the forum is published.
+
+.. image:: help_center/help-center-forum-settings.png
+   :align: center
+   :alt: Overview of a forum's settings page in Odoo Helpdesk.
+
+Karma gains and karma related rights
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When users interact on forums - either through posting questions, submitting answers, or upvoting
+responses - they earn points, called *karma*.
+
+Karma points are used to build the credibility of a user within the community. The more positive
+interactions a user has on the forum, the more karma points they gain. Certain activities, such as
+posting without validation, or editing posts, require users to have earned a specific level of karma
+points to prove their legitimacy and trustworthiness.
+
+These figures can be adjusted on the :guilabel:`Karma gains` and :guilabel:`Karma related rights`
+tabs, located on the forums detail form.
+
+.. tabs::
+
+   .. tab:: Karma gains
+
+      In the :guilabel:`Karma gains` tab, there are listed actions that will cause users to gain
+      (or lose) karma points. Those actions are:
+
+      - Asking a question
+      - Question upvoted
+      - Question downvoted
+      - Answer upvoted
+      - Answer downvoted
+      - Accepting an answer
+      - Answer accepted
+      - Answer flagged
+
+   .. tab:: Karma related rights
+
+      In the :guilabel:`Karma related rights` tab, there are listed activities that users cannot
+      complete, without having a specific level of karma points. Those activities are:
+
+      - Ask questions
+      - Answer questions
+      - Upvote
+      - Downvote
+      - Edit own posts
+      - Edit all posts
+      - Close own posts
+      - Close all posts
+      - Delete own posts
+      - Delete all posts
+      - Nofollow links
+      - Accept an answer on own question
+      - Accept an answer to all questions
+      - Editor features: image and links
+      - Comment on own posts
+      - Comment on all posts
+      - Convert own answers to comments (and vice versa)
+      - Convert all answers to comments (and vice versa)
+      - Unlink own comments
+      - Unlink all comments
+      - Ask questions without validation
+      - Flag a post as offensive
+      - Moderate posts
+      - Change question tags
+      - Create new tags
+
+Create a forum post from a Helpdesk ticket
+------------------------------------------
+
+When a *Helpdesk* team has a *Forum* enabled, tickets submitted to that team can be converted to
+forum posts.
+
+To do that, select a ticket (either from a team's pipeline or from :menuselection:`Tickets --> All
+Tickets` in the :guilabel:`Helpdesk` application).
```

Comment:
I apologize @jero-odoo, I meant a menuselection here not a guilabel in my previous correction.
```suggestion
Tickets` in the :menuselection:`Helpdesk` application).
```